src/quo2/components/links/link_preview/view.cljs

```diff
@@ -0,0 +1,77 @@
+(ns quo2.components.links.link-preview.view
+  (:require [quo2.components.buttons.button :as button]
+            [quo2.components.links.link-preview.style :as style]
+            [quo2.components.markdown.text :as text]
+            [react-native.core :as rn]))
+
+(defn- button-disabled
+  [disabled-text on-enable]
+  [button/button
+   {:before              :i/reveal
+    :size                32
+    :type                :grey
+    :on-press            on-enable
+    :accessibility-label :button-enable-preview}
+   disabled-text])
+
+(defn- description-comp
```

Comment:
Is `-comp` a naming pattern we use for private components?

Reply:
I wouldn't like this to be a pattern @ajayesivan, it's kind of ugly :sweat_smile:, but I had to name them this way because the destructured map keys in the `view` component have the same name as the component, and I prefer to avoid having binding names and function names equal. That's because we can easily shadow one or the other and end up with valid Clojure from the compiler's perspective, but buggy nonetheless. Okay, component tests would probably catch, but why risk right?